rbac/api/cross_access/serializer.py

```diff
@@ -95,3 +95,16 @@ def create(self, validated_data):
         for role in roles:
             request.roles.add(role)
         return request
+
+    def update(self, instance, validated_data):
```

Comment:
This seems never get called, do you mean using super().update but used super().create in the view.py?

Reply:
This should get automatically called any time the user submits a PUT request to the /cross-access-requests/<uuid>/ endpoint using the UpdateModelMixin.

Reply:
Oh, wait, disregard last. Wrong update method. let me re-check how this gets called.